new_instagram/spec/features/pictures_features_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+feature 'pictures' do
+  context 'no pictures added' do
+    scenario 'should display a promp to add a picture' do
+      visit '/pictures'
+      create_user
+      expect(page).to have_content 'No pictures yet'
+      expect(page).to have_link 'Add new picture'
+    end
+  end
+  context "Some pictures added" do
+    before do
+      create_user
+      Picture.create(title: "me")
+    end
+    scenario "display pictures" do
+      visit '/pictures'
+      expect(page).to have_content 'me'
+      expect(page).not_to have_content 'No pictures yet'
+    end
+  end
+
+  context 'creating restaurants' do
+    scenario 'prompts user to fill out a form, then displays the new picture' do
+      visit '/pictures'
+      create_user
+      create_picture
+      expect(page).to have_content 'me'
+      expect(page).to have_selector('.image')
+
+      expect(current_path).to eq '/pictures'
+    end
+  end
+
+  context 'showing restaurants' do
+
+    let!(:me){Picture.create(title: 'ME')}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.